src/components/timeline/LayerActivity.svelte

```diff
@@ -194,6 +201,26 @@
     }
   }
 
+  function removeKeyDownEvent() {
+    if (browser) {
+      document.removeEventListener('keydown', onKeyDown);
+    }
+  }
+
+  function onBlur(e: FocusEvent | undefined) {
+    if (e) {
+      removeKeyDownEvent();
+    }
+  }
+
+  function onKeyDown(event: KeyboardEvent): void {
```

Comment:
Pressing delete multiple times causes the delete warning modal to appear on top of itself successively

Reply:
Thanks for catching that!